csharp/test/Apache.Arrow.Flight.Tests/FlightTests.cs

```diff
@@ -52,6 +53,10 @@ private RecordBatch CreateTestBatch(int startValue, int length)
                 builder.Append(startValue + i);
             }
             batchBuilder.Append("test", true, builder.Build());
+            var keys = new UInt16Array.Builder().AppendRange(Enumerable.Range(startValue, length).Select(i => (ushort)i)).Build();
+            var dictionary = new StringArray.Builder().AppendRange(Enumerable.Range(startValue, length).Select(i => i.ToString())).Build();
+            var dictArray = new DictionaryArray(new DictionaryType(UInt16Type.Default, StringType.Default, false), keys, dictionary);
```

Comment:
Would it be possible to have the test validate the contents of the dictionary?

Reply:
sorry have been out and had other priorities come up ArrowReaderVerifier compares underlying dictionary arrays

https://github.com/apache/arrow/blob/1fd11d33cb56fd7eff4dce05edaba1c9d8a1dccd/csharp/test/Apache.Arrow.Tests/ArrowReaderVerifier.cs#L175-L183